src/models/filter.rs

```diff
@@ -45,12 +45,35 @@ pub struct Filter {
   #[serde(default)]
   #[pyo3(get)]
   not_contains: Vec<TSQuery>,
+  /// AST patterns that should match any subtree of node matching `enclosing_node` pattern
+  #[builder(default = "default_queries()")]
+  #[get = "pub"]
+  #[serde(default)]
+  #[pyo3(get)]
+  contains: Vec<TSQuery>,
```

Comment:
Hmmmm. I guess lets make it unary . It is unclear which query is the `atleast` and `atmost` applicable.